test/functional/search_api_test.rb

```diff
@@ -148,6 +148,34 @@ def app
      assert matcher =~ json
   end
 
+  # search by username and bio, returns users by id when order_by is not provided and sorted direction default DESC
+  test 'search profiles by username and bio without order_by and default sort_direction' do
+    get '/api/srch/profiles?srchString=steff'
```

Comment:
Sorry, looking on my phone... is there also a text for only bio content? Something that has no username matches?

Reply:
we did like this: the previous tests (that we first had in mind to search only by username) have the `field=username` and therefore, returns only the users searched by username. So we added another one that would search only by username, but yeah, good point, we added a new one to search only by bio!

Reply:
@jywarren we pushed some changes, thanks for the feedback!